srml/council/src/lib.rs

```diff
@@ -14,7 +14,188 @@
+//! - **Voting**
+//! 	- [`voting::Trait`](./voting/trait.Trait.html)
+//! 	- [`Call`](./voting/enum.Call.html)
+//! 	- [`Module`](./voting/struct.Module.html)
+//!
+//! ## Overview
+//!
+//! The Council module provides functionality to handle:
+//!
+//! - The voting in and maintenance of council members.
+//! - Proposing, vetoing, and passing motions.
+//!
+//! The Council's role is to represent passive stakeholders. The Council is an on-chain entity comprised of
+//! a set of account IDs. Its primary tasks are to propose sensible referenda and thwart any uncontroversially
+//! dangerous or malicious referenda.
+//!
+//! ### Terminology
+//!
+//! #### Council Proposals (motions.rs)
+//!
+//! - **Council motion:** A mechanism used to enact a proposal.
+//! - **Proposal validity:** A council proposal is valid when it's unique, hasn't yet been vetoed, and
+//! when the council term doesn't expire before the block number when the proposal's voting period ends.
+//! - **Proposal postponement:** Councillors that abstain from voting may postpone a council proposal from
+//! being approved or rejected. Postponement is equivalent to a veto, which only lasts for the cooloff period.
+//! - **Cooloff period:** Period, in blocks, for which a veto is in effect.
+//!
+//! #### Council Proposal Voting (voting.rs)
+//!
+//! - **Proposal:** A submission by a councillor. An initial vote of yay from that councillor is applied.
+//! Unlike the Democracy and Treasury modules, the `Proposal` type is generic and can be anything that
+//! can be dispatched (see [`type Proposal`](./motions/trait.Trait.html#associatedtype.Proposal)).
+//! - **Referendum:** The means of voting on a proposal.
+//! - **Vote:** A vote of yay or nay from a councillor on a single proposal. Councillors may change their vote.
+//! - **Veto:** A council member may veto any council proposal that exists. A vetoed proposal that's valid is set
+//! aside for a cooloff period. The vetoer cannot re-veto or propose the proposal again until the veto expires.
+//! - **Elevation:** A referendum can be _elevated_ from the council to a public referendum. This means it has
+//! been passed to the Democracy module for a public vote.
+//! - **Referendum cancellation:** At the end of a given block we cancel all elevated referenda whose voting period
+//! ends at that block and where the outcome of the vote tally was a unanimous vote to cancel the referendum.
+//! - **Voting process to elevate a proposal:** At the end of a given block we tally votes for expiring referenda.
+//! Referenda that are passed (yay votes are greater than nay votes plus abstainers) are sent to the Democracy
+//! module for a public referendum. If there are no nay votes (abstention is acceptable), then the proposal is
+//! for immediate enactment. Otherwise, there will be a delay period. If the vote is unanimous, then the public
+//! referendum will require a vote threshold of supermajority against to prevent it. Otherwise,
```

Comment:
The supermajority should be quantified.  Is it 2/3, 3/4, or something else?

Reply:
It uses adaptive quorum biasing. Higher level material [here](https://github.com/paritytech/polkadot/wiki/Governance).

It gets implemented in the `VoteThreshold` trait in Democracy. I will add a link to it from here.